galite-tests/src/test/kotlin/org/kopi/galite/demo/web/vaadin/Application.kt

```diff
@@ -44,8 +44,8 @@ fun main(args: Array<String>) {
 }
 
 class GaliteApplication : VApplication(VApplicationTestBase.GaliteRegistry()) {
-  override val sologanImage get() = "slogan.png"
-  override val logoImage get() = "logo_kopi.png"
+  override val sologanImage get() = "ui/vaadin/slogan.png"
```

Comment:
in VLoginBox, setWelcomeImage method can we change the size of the image :
`welcomeImage.setAttribute("height", "10px")`

in VLogo, setImage method can we use image.height to change the size of the image in the header eg : 
`image.height = "40px"`